pkg/operator/controller/dns_status.go

```diff
@@ -169,6 +191,11 @@ func computeDNSProgressingCondition(oldCondition *operatorv1.OperatorCondition,
 		}
 	}
 	if len(messages) != 0 {
+		// if the last status was set to false within the last statusTransitionAllowedDeltaTimeInSeconds, skip the new update
```

Comment:
`statusTransitionAllowedDeltaTimeInSeconds` should be `transitionUnchangedToleration`.